Hardware/Hardware/CardReader/PaymentIntent.swift

```diff
@@ -0,0 +1,24 @@
+/// A PaymentIntent tracks the process of collecting a payment from your customer.
+/// We would create exactly one PaymentIntent for each order
+public struct PaymentIntent: Identifiable {
+    /// Unique identifier for the PaymentIntent
+    public let id: String
+
+    /// The status of the Payment Intent
+    public let status: PaymentIntentStatus
+
+    /// When the PaymentIntent was created
+    public let created: Date
+    ///The amount to be collected by this PaymentIntent, provided in the currency’s smallest unit.
```

Comment:
For great success, please add something like

e.g. USD$5.00 should have amount = 500 and currency = 'usd'